src/components/common.js

```diff
@@ -281,7 +281,7 @@ export default {
         x.roles.includes("zarr-consolidated-metadata")
       );
 
-      if (typeof zarrMetadata !== 'undefined') {
+      if (typeof zarrMetadata !== undefined) {
```

Comment:
The quotes were correct 🤔

Reply:
My bad, was debugging this one in console and totally missed the `typeof` - I'll change this back.